docs/src/languages/ocaml.md

```diff
@@ -0,0 +1,4 @@
+# OCaml
```

Comment:
I see that there's no mac artifact to download from https://github.com/ocaml/ocaml-lsp/releases which means users have to set up ocaml LSP add add to the path themselves, let's mention it in the docs here.

Reply:
I've added some instructions, please give them a shot!